test/SetGoogleSheetIntervalTest.cpp

```diff
@@ -11,6 +11,9 @@ unittest(test) {
   SetGoogleSheetInterval* test = new SetGoogleSheetInterval(tc);
   tc->setNextState(test, true);
 
+  // getCurrentValue
+  assertEqual(65535, test->getCurrentValue());
```

Comment:
Might be nice to add a test for when current value actually has been set.

Reply:
Done.